com.microsoft.mrtk.input/Interactors/HandJointInteractor.cs

```diff
@@ -44,7 +44,7 @@ public abstract class HandJointInteractor :
         /// at which the interaction occurs. This would be the tip of the index finger
```

Comment:
Should we remove `HandNode` and rework `IHandedInteractor.Handedness` in this class too, to match what you did in `PokeInteractor`?

Reply:
Would that be a larger breaking change? I'm good with doing it in this PR or another one for the sake of code quality

Reply:
I misunderstood this comment. Change applied in the same way that it was done with the PokeInteractor.